glTFLoader/glTFLoader.csproj

```diff
@@ -1,7 +1,7 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.3;net35</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.1;net6.0</TargetFrameworks>

```

Comment:
Yes, its really weird that some random overload somewhere just went awol, but I didn't want to bump the TFM because that'd break people.

This doesn't work on .NET Framework. If you judge that to be OK and these TFM's are the lowest you found that build this, there is very little benefit to include .NET 3.1 - its out of support and upgrading that to .NET 6 is straightforward.